hyprshot

```diff
@@ -27,6 +27,9 @@ Options:
   -r, --raw                 output raw image data to stdout
   -t, --notif-timeout       notification timeout in milliseconds (default 5000)
   --clipboard-only          copy screenshot to clipboard and don't save image in disk
+  --border-width            width of the selection border in pixels (default 2)
```

Comment:
You added an option to --help, but it's not implemented?

Reply:
Yup. I absolutely did. Added the border width option because it's not that difficult.